storage/storage.cpp

```diff
@@ -897,4 +904,63 @@ vector<TCountryId> const Storage::GetChildren(TCountryId const & parent) const
 
   return childrenVector;
 }
+
+vector<TCountryId> Storage::GetLocalRealMaps() const
+{
+  std::vector<TCountryId> keys;
+  keys.reserve(m_localFiles.size());
+
+  for(auto const & keyValue : m_localFiles)
+    keys.push_back(keyValue.first);
+  return keys;
+}
+
+vector<TCountryId> Storage::GetDownloadedChildren(TCountryId const & parent) const
```

Comment:
Do you really want to return vector by copy? How about to pass it as an argument by reference and fill in the method?